packages/block-editor/src/components/default-block-appender/style.scss

```diff
@@ -48,7 +48,6 @@
 
 // Ensure that the height of the first appender, and the one between blocks, is the same as text.
 .block-editor-default-block-appender__content {
-	min-height: $empty-paragraph-height / 2;
```

Comment:
I haven't had time to test this yet, are you sure there aren't any negataive consequences of removing this? I'm sure we added it for a reason, back in the day.

Reply:
The reason we added it is to make sure the appender with or without paragraph has the same height. It's broken with or without this style now.

Reply:
Not sure I follow but I'll check when I have a moment.